### ISSUE TITLE
Failed to create RDS with databases

### ISSUE THREAD

**Issue by RainFlying** [alibaba/terraform-provider#498]
What in my tf file:
```
resource "alicloud_db_account" "rds_user" {
  instance_id = "${alicloud_db_instance.rds.id}"
  name = "${var.db_user}"
  password = "${var.db_password}"
  type = "Normal"
}

resource "alicloud_db_database" "default" {
  instance_id = "${alicloud_db_instance.rds.id}"
  name = "${var.instance_name}-placeholder"
  character_set = "utf8mb4"
}

resource "alicloud_db_database" "dscp" {
  instance_id = "${alicloud_db_instance.rds.id}"
  name = "dscp"
  character_set = "utf8mb4"
}

resource "alicloud_db_database" "try_dscp" {
  instance_id = "${alicloud_db_instance.rds.id}"
  name = "try_dscp"
  character_set = "utf8mb4"
}

resource "alicloud_db_account_privilege" "default" {
  instance_id = "${alicloud_db_instance.rds.id}"
  account_name = "${alicloud_db_account.rds_user.name}"
  privilege = "ReadWrite"
  db_names = ["${alicloud_db_database.default.name}", "${alicloud_db_database.dscp.name}", "${alicloud_db_database.try_dscp.name}"]
  depends_on = ["alicloud_db_account.rds_user", "alicloud_db_database.dscp", "alicloud_db_database.try_dscp", "alicloud_db_database.default"]
}
```

The first "terraform apply" showed the following error message:
```
* alicloud_db_account_privilege.default: 1 error(s) occurred:

* alicloud_db_account_privilege.default: Grant Account dscp Privilege ReadWrite got an error: &errors.errorString{s:"Grant DB test-placeholder account dscp privilege got an error: &errors.ServerError{httpStatus:403
, requestId:\"C47C44A7-7B9F-45CD-9219-6A60B465D5A1\", hostId:\"rds.aliyuncs.com\", errorCode:\"OperationDenied.DBStatus\", recommend:\"\", message:\"The operation is not permitted due to status of the database.\",
comment:\"\"}."}
```
Tried the second time, the resources would be created successfully.

**Comment by RainFlying** [comment]
Running terraform provider version 1.9.3.

**Comment by xiaozhu36** [comment]
HI @RainFlying It has been fixed by v1.9.6+ (https://github.com/alibaba/terraform-provider/pull/534/files) and I have made many tests for this error. Please check it.

In addition, I will close this issue and if your problem still exists, please reopen it.
